Controller/Adminhtml/Integration/CreateStore.php

```diff
@@ -268,14 +257,13 @@ private function cleanCache()
     }
 
     /**
-     * Get Process Callback URL
-     *
-     * @param StoreInterface $store
-     *
-     * @return string
+     * We obtain the url associated with the main language search_engine
      */
-    private function getProcessCallbackUrl(StoreInterface $store): string
-    {
-        return $store->getBaseUrl() . 'doofinderfeed/setup/processCallback?storeId=' . $store->getId();
+    function get_primary_site_url_in_se($search_engines, $primary_language) {    
```

Comment:
If this function is only used here, I would change it to private :smile: 
```suggestion
    private function get_primary_site_url_in_se($search_engines, $primary_language) {    
```